draft-bft-rats-kat.md

```diff
@@ -119,22 +119,38 @@ in {{Section 8 of -cbor}} and {{Appendix G of -cddl}}.
 
 # Architecture
 
-In an architecture for key attestation there are three or more parties communicating with
-each other. From a key attestation point of view, there are the Key Attestation Service,
-the Presenter and the Recipient. Additional parties are added when considering attestation-
-related entities.
+Key attestation is an extension to the attestation functionality described in {{-rats-arch}}. 
```

Comment:
It should be better termed as "Augmented Key Attestation" or "Enhanced Key Attestation"

Reply:
not sure, we are enhancing or augmenting key attestation